crd-generator/README.md

```diff
@@ -11,7 +11,7 @@ Please give it a try and report any issue you might find.
 - Several annotations are available to override the default behavior if it doesn't fit your needs
 - Validation schema is automatically derived from your class' spec implementation (or from the CR
   class, if no spec is present)
-- CRDs are automatically generated at compile-time if you include the `io.fabric8:crd-generator`
+- CRDs are automatically generated at compile-time if you include `io.fabric8:crd-generator-api` 
```

Comment:
This still isn't correct. The dependency to use is `crd-generator-apt` if you want the CRD automatically generated at compile-time.